### ISSUE TITLE
Binding on a Fluent.TextBox Text property does not work.

### ISSUE THREAD

**Issue by polepage** [fluentribbon/Fluent.Ribbon#361]
When Binding a string property on the Text property of a Fluent.TextBox the binding does not work. The setter of my property is never called when typing in the box or setting focus elsewhere. I tried to change the UpdateSourceTrigger to LostFocus and the Mode to TwoWay on the Binding (even though the are the default values) to no avail.

Using snoop I can see that the underlying System.Windows.Controls.TextBox is modified when I type, but the Fluent.TextBox is not moved by any of it.

--
### Environment

- Fluent.Ribbon 5.0.0
- Windows 10
- .NET Framework 4.6.1

**Comment by polepage** [comment]
I tried with latest release on nuget. This actually worked before 5.0.0.

**Comment by polepage** [comment]
We found we have this issues with other properties such as IsReadOnly.

I investigated the issue in the showcase code. The culprit seems to be the simplification bindings forwarding.

In the constructor, and more importantly in the OnApplyTemplate method, you replaced the Bind list by a call to the new method ForwardBindings. However this breaks a few bindings. The original bindings used the DependencyProperty definitions in TextBoxBase or System.Windows.Controls.TextBox, while the ForwardBindings method use the DependencyProperty definitions of Fluent.TextBox.

I coded a work around in my own code (overrided OnApplyTemplate, called base then re-added bindings as they were). It is obviously not a good solution, but it make my code works for now.

I'm not sure of the implications of those changes, but my solution would be to revert the use of the old (and different in each case) list of Bind calls, with the "nameof" amelioration. If I have some time, I'll make a merge request with that modification, unless you tell me there is something more to it.
